code/datums/components/tippable.dm

```diff
@@ -116,6 +116,8 @@
 		CRASH("Tippable component: do_tip() called with QDELETED tipped_mob!")
 
 	to_chat(tipper, span_warning("You tip over [tipped_mob]."))
+	if (!isnull(tipped_mob.client))
+		log_attack("[tipper] ([tipper.key]) has tipped over [tipped_mob] ([tipped_mob.key])!")
```

Comment:
1. Use `key_name`
2. Change to `log_message` with `LOG_ATTACK` for both tipper and tipped.